keyboards/rarepotato8de/3x3macropad/info.json

```diff
@@ -0,0 +1,69 @@
+{
+    "manufacturer": "RarePotato8DE",
+    "keyboard_name": "3x3macropad",
+    "maintainer": "RarePotato8DE",
+    "bootloader": "caterina",
+    "build": {
+        "lto": true
+    },
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "F4", "pin_b": "B1"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "combo": false,
+        "encoder": true,
+        "extrakey": true,
+        "key_override": false,
+        "nkro": true,
+        "oled": true,
```

Comment:
If OLEDs are enabled at the keyboard level, there needs to exist some code for them as well at the keyboard level in a `3x3macropad.c` file.